db/seeds.rb

```diff
@@ -155,3 +155,63 @@
   compte.structure = structure_eva
   compte.password = SecureRandom.uuid
 end
+
+genre = QuestionQcm.find_or_create_by(nom_technique: 'genre') do |question|
+  question.libelle = 'Genre'
+  question.intitule = 'Vous êtes ?'
+  question.choix = [
+    Choix.find_or_create_by(nom_technique: 'homme', intitule: 'Un homme', type_choix: 'bon'),
+    Choix.find_or_create_by(nom_technique: 'femme', intitule: 'Une femme', type_choix: 'bon'),
+    Choix.find_or_create_by(nom_technique: 'autre', intitule: 'Autre', type_choix: 'bon')
+  ]
+end
+
+langue_maternelle = QuestionQcm.find_or_create_by(nom_technique: 'langue_maternelle') do |question|
+  question.libelle = 'Langue maternelle'
+  question.intitule = 'Le français est-il votre langue maternelle ?'
+  question.choix = [
+    Choix.find_or_create_by(nom_technique: 'oui', intitule: 'Oui', type_choix: 'bon'),
+    Choix.find_or_create_by(nom_technique: 'non', intitule: 'Non', type_choix: 'bon'),
+  ]
+end
+
+lieu_scolarite = QuestionQcm.find_or_create_by(nom_technique: 'lieu_scolarite') do |question|
+  question.libelle = 'Lieu de scolarite'
+  question.intitule = "Etes-vous allé à l'école ?"
+  question.choix = [
+    Choix.find_or_create_by(nom_technique: 'france', intitule: 'Oui, en France', type_choix: 'bon'),
+    Choix.find_or_create_by(nom_technique: 'etranger', intitule: 'Oui, dans un autre pays', type_choix: 'bon'),
+    Choix.find_or_create_by(nom_technique: 'non_scolarise', intitule: 'Non', type_choix: 'bon'),
+  ]
+end
+
+niveau_etude = QuestionQcm.find_or_create_by(nom_technique: 'niveau_etude') do |question|
+  question.libelle = "Niveau d'étude"
+  question.intitule = "Quel niveau d'étude avez-vous atteint ?"
+  question.choix = [
+    Choix.find_or_create_by(nom_technique: 'pas_etudie', intitule: "Je ne suis pas allé à l'école", type_choix: 'bon'),
+    Choix.find_or_create_by(nom_technique: 'college', intitule: 'Niveau Collège', type_choix: 'bon'),
+    Choix.find_or_create_by(nom_technique: 'cfg_dnb', intitule: 'Niveau certificat de formation générale ou diplome national du brevet', type_choix: 'bon'),
+    Choix.find_or_create_by(nom_technique: 'cap_bep', intitule: 'CAP/BEP', type_choix: 'bon'),
```

Comment:
Je ne sais pas si c'est fait exprès d'avoir retiré "Niveau" par rapport au document de référence pour les intitulés des quatre réponses suivantes ou si c'est un oubli ?
```suggestion
    Choix.find_or_create_by(nom_technique: 'cap_bep', intitule: 'Niveau CAP/BEP', type_choix: 'bon'),
```